assets/openobserve/search.py

```
@@ -0,0 +1,45 @@
+import base64
+import requests
+import sys
+import json
+import time
+
+#https://openobserve.ai/docs/ingestion/logs/python/#python
+
+user = "root@clpbench.com"
+password = "password"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "password"

(S105)